.github/workflows/check-trivy.yaml

```diff
@@ -65,6 +65,6 @@ jobs:
 
       - name: Upload Trivy scan results to GitHub Security tab
         if: ${{ always() }}
-        uses: github/codeql-action/upload-sarif@0116bc2df50751f9724a2e35ef1f24d22f90e4e1 #v2.22.3
+        uses: github/codeql-action/upload-sarif@65c74964a9ed8c44ed9f19d4bbc5757a6a8e9ab9 #v2.16.1
```

Comment:
could you please recheck this? The latest release maybe called CodeQL Bundle, but under tags the other can be seen as well. In the repo they also say that version 2 is deprecated and will not get supported starting December 2024. Maybe it is good if we already use v3? See https://github.com/github/codeql-action?tab=readme-ov-file#supported-versions-of-the-codeql-action

Reply:
codeql-action = v3.24.2
https://github.com/github/codeql-action/releases/tag/v3.24.2